src/control/server/instance.go

```diff
@@ -259,7 +259,7 @@ func (ei *EngineInstance) handleReady(ctx context.Context, ready *srvpb.NotifyRe
 		ei.log.Error(err.Error()) // nonfatal
 	}
 
-	r, localJoin, err := ei.determineRank(ctx, ready)
+	r, localJoin, map_version, err := ei.determineRank(ctx, ready)
```

Comment:
nit: This will cause linter errors, but only in editors. Go variables are `camelCase`. I am OK with landing as-is, but would prefer to see this updated soon.

Reply:
Errrh, sorry, this is a stupid overlook of mine---I haven't written any Go for a while. I'll submit a separate patch to rename it.